our tar fails to extract this because of a bunch of self-referential hardlinks apparently
probably because it's a tar of /sys
--- a/test/sysfs/setup.sh
+++ b/test/sysfs/setup.sh
@@ -6,6 +6,6 @@
 TESTDIR="$BASEDIR/$(basename -s .tar.xz ${TARFILE})"
 
 mkdir -p "${TESTDIR}"
-tar -x -f "${TARFILE}" -C "${TESTDIR}" || exit 1
+gtar -x -f "${TARFILE}" -C "${TESTDIR}" || exit 1
 
 echo "${TESTDIR}"
